core/src/main/java/com/capitalone/dashboard/model/ServiceStatus.java

```diff
@@ -4,14 +4,36 @@
  * Denotes the status of a product or service
  */
 public enum ServiceStatus {
-    Ok, Warning, Alert;
+    Ok(200, 200),
+    Warning(300, 400),
+    Unauth(401, 401),
```

Comment:
@nireeshT @rob-miller-777 This url should return a 401 from GitHub: https://api.github.com/?access_token=OAUTH-TOKEN